docker-compose.yml

```diff
@@ -1,34 +1,33 @@
 version: "3.8"
 services:
-    web:
-        build: .
-        ports:
-            - "8000:8000"
-        command: npx nodemon -L app.js
-        volumes:
-            - ./app:/usr/src/app
-            - /usr/src/app/node_modules
-            - ./.git/FETCH_HEAD:/usr/src/app/FETCH_HEAD
-        env_file: 
-            - .env
-        depends_on:
-            - mongo
-        
-    mongo:
-        image: mongo:latest
-        container_name: mongo_db
-        volumes:
-            - ./config/init.sh:/docker-entrypoint-initdb.d/init.sh
-            - ./config/mongod.conf:/etc/mongod.conf
-            - ./logs:/var/log/mongodb/
-            - /data/db
-        env_file:
-            - .env
-        ports:
-            - "27017:27017"
-        restart: on-failure:5
-        command: ["mongod", "-f", "/etc/mongod.conf"]
+  web:
+    build: .
+    ports:
+      - "8000:8000"
+    command: npx nodemon -L app/app.js
+    volumes:
+      - ./app:/usr/articleblog/app
+    environment:
+      - COMMIT_HASH
+    env_file:
+      - .env
+    depends_on:
+      - mongo
+
+  mongo:
+    image: mongo:latest
+    container_name: mongo_db
+    volumes:
+      - ./config/init.sh:/docker-entrypoint-initdb.d/init.sh
+      - ./config/mongod.conf:/etc/mongod.conf
+      - ./logs:/var/log/mongodb/
+      - db-data:/data/db
```

Comment:
Why do we bring back mapping? Won't it cause issues again?

Reply:
See https://github.com/Jordan141/articleblog/pull/259

Reply:
It is a named volume, not a mapped one, and as such it will behave in a similar way to an anonymous volume. It is not mapped to a specific directory on host's filesystem.

The main reason I've added it because without, `clear-mongodb.sh` script is broken.
In fact, the `clear-mongodb.sh` script in #259 is broken, because without assigning `db-data` name on the volume, Docker will create a volume with a random name, and the script expects a predictable name:
```
docker volume rm articleblog_db-data >&3
```
In #259, docker-compose will create 3 volumes:
1. a random name for /data/db
2. a volume named db-data (articleblog_db-data) that will be left unused because it is never assigned to a mount point inside a container
3. a volume named mongodb-config (articleblog_mongodb-config) that will be unused too

The existence of 2. and 3. are caused by the definition at the end of docker-compose:
```yaml
volumes:
    db-data:
    mongo-config:
````
MongoDB will write data to the 1. volume with random name. The `clear-mongodb.sh` script will remove `articleblog_db-data` volume successfully, but it is an empty volume that isn't used by MongoDB at all.

This PR keeps the:
```yaml
volumes:
    db-data:
````
But assigns it to container's mount point:
```yaml
      - db-data:/data/db
```
So that MongoDB will write to `articleblog_db-data` and not to another volume with a random name.